Assets/Code/Interactables/Effects/Dynamic Effects/Interacts/CreatePassiveEffect.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+
+public class CreatePassiveEffect : DynamicEffect {
+    // DynamicEffect that creates the specified PassiveStaticEffect (Status only)
+    // These are specifically for effects that do not target
+
+    public CreatePassiveEffect(int effectCount) : base(effectCount) {
+    }
+
+    public override void ResolveEffect() {
+        Debug.Log("amount:" + effectCount.ToString());
+        for (int i = 0; i < effectCount; i++) {
+            CardManager.SharedInstance.DrawCard();
```

Comment:
is this temp code?

Reply:
Yeah it's all temp

Reply:
maybe add a TODO here as well then? since we don't have tickets :P